tests/operators/functionals/test_mse_functional.py

```diff
@@ -12,15 +12,13 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-
 import pytest
 import torch
-
-from mrpro.operators.functionals._mse_data_discrepancy import mse_data_discrepancy
+from mrpro.operators.functionals._mse_data_discrepancy import MSEDataDiscrepancy
 
 
 @pytest.mark.parametrize(
-    'data, x, expected_mse',
+    ('data', 'x', 'expected_mse'),
```

Comment:
If we use `MSEDataDiscrepancy` should we then not also use `expected_MSE` (which in my opinion looks ugly...)